### PR TITLE
Facebook 1.0.13

### PR DESCRIPTION
**Pull request by JohnNilsson**
This PR is based on the facebook-1.0.11 PR from before (including source-map-loader). Pretty straight forward merge resolutions.

As before, tested insofar that "yarn build" and "yarn start" seem to work.

(I also got around my previous issue with minification by using `"downlevelIteration": true` i tsconfig. So perhaps no need for babel after all)

### CODE REVIEW

**Comment by JohnNilsson** [comment]
If the Travis build needs a workaround it should work by pulling in https://github.com/facebookincubator/create-react-app/commit/e8b58edf637ee07b44492b715814797eaf272ce2 , left it out since it wasn't part of the 1.0.13 tag

**Comment by gyzerok** [comment]
@wmonk it is probably very scary for you to review "merging upstream" PRs due to huge amount of stuff you need to dig into. Your `react-scripts` fork is what helps me and according to `npm` many others to enjoy benefits of Create React App and TypeScript altogether.

I am really awaiting upstream since `1.0.11` to be merged due to the performance improvements it is promising. Could you, please, spare some time and make it happen? :)

**Comment by wmonk** [comment]
Hi all, I will look to get this out over the weekend. Thanks for your patience!

**Comment by wmonk** [comment]
@JohnNilsson I will be more confident if CI is passing. Could you pull in the necessary commits, and I will release a backdated commit, to match `1.0.13`.

File: packages/react-scripts/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-scripts-ts",
-  "version": "2.6.0",
+  "version": "1.0.12",
```
Comment (wmonk) [inline packages/react-scripts/package.json]: Please change version back to `2.6.0`

**Comment by wmonk** [comment]
Woohoo 🎉 thanks @JohnNilsson really appreciated 👍

**Pull request merged**